client/me/reauth-required/index.jsx

```diff
@@ -148,14 +152,21 @@ module.exports = React.createClass( {
 	},
 
 	render: function() {
-		var codePlaceholder = '123456';
+		var codePlaceholder = this.translate( 'e.g. 123456', {
+			context: '6 digit two factor code placeholder.',
+			textOnly: true
+		} );
 
 		if ( this.props.twoStepAuthorization.isTwoStepSMSEnabled() ) {
-			codePlaceholder = '1234567';
+			codePlaceholder = this.translate( 'e.g. 1234567', {
+				context: '7 digit two factor code placeholder.',
+				textOnly: true
+			} );
 		}
 
 		return (
 			<Dialog
+				autoFocus={ false }
```

Comment:
In my testing it is necessary.

My understanding is the the `Dialog` component auto focuses on the content by default. So passing in `autoFocus={false}` short circuits that and allows us to set our own `autoFocus`.

See [this block](https://github.com/Automattic/wp-calypso/blob/master/client/components/dialog/dialog-base.jsx#L36-L38).